### ISSUE TITLE
用久了 会导致浏览器很卡[BUG]

### ISSUE THREAD

**Issue by wk849411933** [xxxily/h5player#632]
1. **问题描述**
用久了 会导致浏览器很卡

2. **出现上述问题的地址**

3. **你使用的浏览器与浏览器版本信息**
edge和chrome都会变卡

4. **其他附注信息**
我 测试过开启脚本后，使用一段时间，比如两小时，会发现浏览器点击会有明显的卡顿，禁用插件重启后就不会这样。

**Comment by JinLong369** [comment]
确实。我用的chrome，B站、youtube等视频网站的视频播放页开的时间长一些，页面就会变卡，然后开启新的标签页（哪怕只是百度）就会一直加载，一定概率会导致浏览器崩溃。一般都得重启浏览器才能恢复正常。

主要经常用B站看一些教程，都是很多集的那种，每次只要多看几集，浏览器必然变卡然后重启，很影响体验……

希望能尽量修复吧
